presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -47,7 +48,10 @@ public class IcebergConfig
     private double statisticSnapshotRecordDifferenceWeight;
 
     private HiveStatisticsMergeStrategy hiveStatisticsMergeStrategy = HiveStatisticsMergeStrategy.NONE;
-
+    private boolean manifestCachingEnabled;
+    private long maxManifestCacheSize = CatalogProperties.IO_MANIFEST_CACHE_MAX_TOTAL_BYTES_DEFAULT;
+    private long manifestCacheExpireDuration = CatalogProperties.IO_MANIFEST_CACHE_EXPIRATION_INTERVAL_MS_DEFAULT;
+    private long manifestCacheMaxContentLength = CatalogProperties.IO_MANIFEST_CACHE_MAX_CONTENT_LENGTH_DEFAULT;
```

Comment:
Suggest directing importing the constant variables. E.g.
`CatalogProperties.IO_MANIFEST_CACHE_MAX_TOTAL_BYTES_DEFAULT` => `IO_MANIFEST_CACHE_MAX_TOTAL_BYTES_DEFAULT`